---
title: "Group 28 presentation"
format: 
  revealjs:
    theme: simple
    transition: fade
    embed-resources: true
author: 
- "Iben Sommerand - s2035"
- "Jonas Sennek - s2035"
- "Emilie Wenner - s"
- "Torbj√∏rn Bak Regueira - s203555"
- "Vedis Arntzen - s203546"
editor: visual

---

## Introduction

```{r}
#| echo: false
#| eval: true
#| message: false
#| output: false
library(tidyverse)
library(patchwork)
library(scales)
library(ggridges)
library(patchwork)
library(ggseqlogo)
library(dplyr)
library(table1)
library(purrr)
library(broom)
library(here)
```

```{r}
source(file = here("R/99_proj_func.R"))
```


## Materials and method

-   The analysis was performed on the dataset "GDC TCGA Breast Cancer (BRCA)"
-   We included data on gene expression RNAseq and phenotype

```{r}
#| output: false
raw_dir <- here("data/_raw/")
data_loc <- "https://gdc-hub.s3.us-east-1.amazonaws.com/download/"
download_file(raw_dir, 
              data_loc, 
              data_file = "TCGA-BRCA.star_fpkm.tsv.gz", 
              new_file_name = "gene_expression_fpkm.tsv.gz")
```

Materials: What data did you use and where did you get it from? Methods: Which modelling did you use? Think of the methods section as a recipe for how to go from raw to results =\> Flow chart?

## Descriptive analysis


## Analysis 1: Cancer stage
:::: {.columns}

::: {.column width="50%"}
```{r, echo=FALSE, fig.cap='Figure 2: Survival time by Cancer Stage'}
knitr::include_graphics(path = here("results/05_01_years_until_death_by_cancer_stage.png"))
```
:::

::: {.column width="50%"}
```{r, echo=FALSE, fig.cap='Figure 3: Vital Status by Cancer Stage'}
knitr::include_graphics(path = here("results/05_02_vital_status_by_cancer_stage.png"))
```
:::

::::

## Analysis 2: PCA

## Analysis 3: Linear model


## Analysis 4: 

## Discussion
